core/Plugin.php

```diff
@@ -451,6 +451,19 @@ public static function getPluginNameFromNamespace($namespaceOrClassName)
     }
 
     /**
+     * Override this method in your plugin class if you want your plugin to be loaded during tracking.
+     *
+     * Note: If you define your own dimension or handle a tracker event, your plugin will automatically
+     * be detected as a tracker plugin.
+     *
+     * @return bool
+     */
+    public function isTrackerPlugin()
```

Comment:
RequestProcessors are defined in DI so they can exist in any namsepsace. So they can't be detected.

The method is used in the new plugin that requires this change. It could be moved to plugin.json, that would be cleaner I think.